### DISCUSSION TITLE
[Feature request] Which is which plugin in feed render

### DISCUSSION THREAD

**Discussion by jennydaman** [FNNDSC/ChRIS_ui#792] [Ideas]
I do experiments in _ChRIS_ feeds. Here's a vague description: starting with a dataset, I run some ChRIS plugins to play around with processing the data in different ways, then after each "data processing" plugin, I run two "quality control" plugins.

![image](https://user-images.githubusercontent.com/20404439/212320659-c21b8a94-d309-47fe-a87b-a959cf5bed3b.png)

At a glance, it's hard for a user to tell what's what. For instance, if a collaborator wanted to check the results of every instance of [pl-smoothness-error](https://github.com/fnndsc/pl-smoothness-error), they would have to do a "brute-force" search by clicking on every single circle.

What are some ways we can improve this use case?

**Comment by jennydaman** [comment]
Next to the toggle switches in the top-left corner, add a toggle which color-codes nodes by plugin ID instead of by status.

The color coding can be decided by a mapping of plugin ID to color, like how compute resources are color-coded in pipeline customization.

![image](https://user-images.githubusercontent.com/20404439/212322041-0b912b23-04d7-4060-ab93-0195f6a0dd88.png)

Actually, maybe it should be a drop-down: color-coding by status, compute env, plugin, compute time, output size?

**Comment by jennydaman** [comment]
Alternative idea: have a search bar overlayed in the feed view, which highlights the nodes which match the search (and dims the rest of them).

**Reply by Quintinus98** [reply]
This is a nice idea
